frontend/src/components/GardenctlInfo.vue

```diff
@@ -0,0 +1,56 @@
+<!--
+SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and Gardener contributors
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+<template>
```

Comment:
I think it makes sense to use this component also for the account kubeconfig

<img width="1039" alt="Screenshot 2023-01-13 at 18 03 12" src="https://user-images.githubusercontent.com/35373687/212378990-527ad56f-c095-4629-82ad-b9b4ea0d9009.png">

Maybe not in this PR but we could create an issue for it